route/router.go

```diff
@@ -117,8 +117,8 @@ type Stop struct {
 
 // TimeWindow represents a time window for a shift of a vehicle or a stop.
 type TimeWindow struct {
-	Start time.Time `json:"start"`
-	End   time.Time `json:"end"`
+	Start time.Time `json:"start,omitempty"`
+	End   time.Time `json:"end,omitempty"`
```

Comment:
Out of curiosity: why is `omitempty` required on input structs? Isn't that only relevant for `Marshal` and not so much `Unmarshal`.

Reply:
The json schema validation library marks fields with omitempty  as optional. And as you correctly say, _we_ only use it on outputs for marshaling via the json package in the standard library. So this should not interfere.

Reply:
Ah, makes sense. Thanks for clarifying.